frontend/src/assets/scss/form/switch.scss

```diff
@@ -6,10 +6,10 @@
     --el-switch-off-color: #d1d5db; // Gray-300
   }
 
-  @apply flex grow justify-between items-center ml-4 cursor-pointer;
+  @apply flex grow-0 justify-between items-center ml-4 cursor-pointer;
 
   // Label
   & .el-switch__label {
-    @apply text-xs text-gray-900 font-medium #{!important};
+    @apply text-xs text-gray-900 mr-2 font-medium #{!important};
```

Comment:
This will affect all toggles we have on the app. 
Can you make these changes on `frontend/src/shared/modules/filters/components/partials/FilterIncludeSwitch.vue`?
Here is the updated <style> of the component:
```
<style lang="scss">
.filter-include-switch{
  @apply grow-0;

 .el-switch__label{
    @apply mr-2;

   font-weight: normal !important;
 }
}
</style>

```

Reply:
thanks 👍🏽 , updated.